assets/src/dashboard/app/views/editorSettings/index.js

```diff
@@ -196,6 +203,45 @@ function EditorSettings() {
           </Main>
         </Layout.Scrollable>
       </Wrapper>
+
+      {isActiveRemoveLogoDialog && (
+        <Dialog
+          isOpen={true}
```

Comment:
Can the `isActiveRemoveLogoDialog` be used in this `isOpen` prop to control the visibility of the `Dialog`?

Reply:
I updated so that `isActiveRemoveLogoDialog` is always a boolean to control `isOpen` and eliminate the need to check if there's a value before rendering the dialog on the view. Nice clean up, thanks for suggesting.